overlord/state/change.go

```diff
@@ -175,6 +175,11 @@ func (c *Change) SetStatus(s Status) {
 	c.status = s
 }
 
+// State return the pointer to the global State
```

Comment:
It's not really global. I'd word this differently. Perhaps ``change specific State``